Project.toml

```diff
@@ -27,7 +27,7 @@ Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 TypedTables = "9d95f2ec-7b3d-5a63-8d20-e2491e220bb9"
 
 [compat]
-BlockArrays = "0.5, 0.6, 0.7, 0.8, 0.9, 0.10, 0.11"
+BlockArrays = "0.11"
```

Comment:
I have no problem here, but I had to laugh a little bit because just a few months ago we were blocking on being incompatible with BlockArrays 0.11.